### DISCUSSION TITLE
Unable to Send Test Label

### DISCUSSION THREAD

**Discussion by zwillard8** [porrey/Virtual-ZPL-Printer#65] [Q&A]
Good morning everyone,

I am new to this Zebra Printer Emulator but I have watched 2 videos that guided me on how to install/setup the printer. When I installed the emulator, everything was set to Generic (network card, driver, etc.) and the IP address was set to 127.0.0.2 and the Port was set to 9100.

The Printer Configuration within the emulator has the IP address and the port set to 127.0.0.2 and 9100 respectively. 

When I select the Printer configuration with these settings and then start the printer. I want to test the printer so I select Test in the bottom right hand corner of the screen. I select the label to print and select Send and absolutely nothing happens regardless of how many times I select the Send button. 

Does anyone have any advice on how to resolve this problem.

![Virtual ZPL Printer Issue](https://github.com/user-attachments/assets/05cae9a0-db2b-4b94-b9d6-85303712ed74)

**Comment by rbrinker155** [comment]
I have the exact same issue, did you ever find a way around this? I have tried running the ZPL Printer as admin and still the same issue.

**Comment by CMIGIT** [comment]
Hello,

Have you try with 127.0.0.1 ?

Maybe...

**Comment by rafamuz** [comment]
Try with 0.0.0.0 too...
